src/pages/en.md

```diff
@@ -316,6 +316,10 @@ For example, Rails has a [`rails-dev-box`](https://github.com/rails/rails-dev-bo
 
 [Docker](https://www.docker.com) is also a good way to make dependency management simpler.
 
+- [ ] Promptly update version of gem at RubyGems
```

Comment:
I think, for now it would be enough to have the only one publishing-related item under the "Misc" section.

Project _management_ itself is not a part of the checklist–it fits [https://opensource.guide](https://opensource.guide) more. And as its stated in the very beginning of the checklist–we don't want to re-invent the existing guide.

From the other side, providing both general and Ruby-specific tips on how to make the code, say, more "accessible"–that's what we (okay, at least me) want.

For example, it make sense to add a note about publishing thru CI (e.g. [Travis](https://docs.travis-ci.com/user/deployment/rubygems/)).